documentation/staging/1.4/content/navigate/kubernetes/k8s-weblogic-domain.md

```diff
@@ -0,0 +1,268 @@
+that resource into a Kubernetes cluster.
+
+The `Domain UID` field defines the name of the Kubernetes Domain custom resource object.  This name must be unique
+within the Kubernetes namespace where it will be created, which is controlled by the `Kubernetes Namespace` field.
+
+The default value of the `Domain UID` field is based on the WebLogic domain's name, as defined by the WDT model:  
+
+- When using either "Model in Image" or "Domain in Image" [domain location]({{< relref "/navigate/project-settings#choosing-a-domain-location" >}}),
+the `Domain Home Path` field is read-only and its value is set using the `Domain Home Directory` field under `Advanced`.  
+- When using "Domain in PV", this field must be set to the fully
+qualified path to the domain home directory in the persistent volume.  For example, if the persistent volume mount
+path is set to `/shared` and the domain home is located in `/domains/mydomain` in the persistent volume, then the
+`Domain Home Path` field must be set to `/shared/domains/mydomain`.
+
+For "Model in Image", use the `Domain Type` field to specify the type of domain to create using the model.
+
+Set the Kubernetes secret name where the WebLogic domain credentials will be stored using the
+`WebLogic Credentials Secret Name` field.  Set the value of the WebLogic Server administration credentials using the
+`WebLogic Admin Username` and `WebLogic Admin Password` fields.  Remember that these fields must be set appropriately
+for the selected domain location:
+
+- For "Model in Image", these fields are used to set the WebLogic Server credentials used to create the domain, and
+  by the operator at runtime to perform actions on the domain (for example, start a server).
+- For "Domain in Image", the domain was created using the WebLogic Image Tool using the credentials specified in the
+  model.  These values must match the ones used during the image creation process.
+- For "Domain in PV", the domain was created outside the application so the values provided in these fields must match
+  the ones used when the domain was created.
+
+When using "Domain in PV", three additional fields associated with the persistent volume will be shown:
+
+- `Persistent Volume Name` - The name of the persistent volume to use in the Domain resource file.
+- `Persistent Volume Claim Name` - The persistent volume claim to associate with the persistent volume.
+- `Persistent Volume Mount Path` - The path to the persistent volume within the container(s).
+- `Enable Log Home` - Enables the ability to separate the logs directory from the domain directory on the persistent
+  volume.
+- `Log Home Path` - The path to use for the log home when `Enable Log Home` is enabled.
+
+The `WebLogic Kubernetes Operator Installed Version` field assumes an installed WebLogic Kubernetes Operator version 4.0.
+If you used WKTUI to install the WebLogic Kubernetes Operator, the field will display its version. If not, you can get the
```

Comment:
I think we should be careful with the wording here.  We need to emphasize that the install process will populate the field if the install is done by WKTUI 1.4.  I would say something like:

If you use WKTUI to install the WebLogic Kubernetes Operator, the field will be populated with the installed version.